### PR TITLE
First edit.

### PR DESCRIPTION
**Pull request by hassan-obeid**

### CODE REVIEW

File: notebooks/_01-ProblemStatement.md [hunk 1]
```diff
@@ -19,4 +19,4 @@ jupyter:
 ## What is the problem motivating this project?
 
 
-This project attempts to bridge the gap between causal inference modeling practices and the field of demand modeling (discrete choice modeling) in transportation by demonstrating an approach to think causally about the modeling problem.
+This project will attempt to bridge the gap between causal inference modeling practices and the field of demand modeling (discrete choice modeling) in transportation by demonstrating an approach to think causally about the modeling problem.
```
Comment (timothyb0912) [inline notebooks/_01-ProblemStatement.md]: The prompt is to note what problems the project aims to solve, not what we will do to solve the problem.

Try to frame the response in terms of "___ is wrong" or "___ needs to be improved."

File: notebooks/_02-ProjectRationale.md [hunk 2]
```diff
@@ -19,4 +19,4 @@ jupyter:
 ## Why work on this problem now? 
 
 
-The field of causal inference has had a significant boost in interest recently with a new direction of research based on both the potential outcome framework and the double ML methods, as well as causal graphical models put forward by Pearl.
+The field of causal inference has had a significant boost in interest recently with a new direction of research based on the potential outcome framework and the double ML methods, as well as causal graphical models put forward by Pearl.
```
Comment (timothyb0912) [inline notebooks/_02-ProjectRationale.md]: Be more specific please? What recent research makes it such that this project is a good idea to work on?

File: notebooks/_03-Vision.md [hunk 3]
```diff
@@ -18,8 +18,7 @@ jupyter:
...
+We will demonstrate using simulations how to structurally approach demand modeling and modeling in general while paying attention to the causal structure of how people make their choices, and the data generation process in general. 
 We will build on Brathwaite’s work and review paper and demonstrate with steps how to incorporate and think about causal inference in the context of DCM. We will also demonstrate the value of such an approach by contrasting the outcome of this way of thinking to that of traditional demand modeling where little care is given to the causal structure. 
 
 Specifically, there will be two main deliverables in our work/presentation. The first one is a simple application of the steps outlined in Brathwaite et. al to a simulation example, where we will also show what would happen to our causal estimates under different assumptions of the causal graph. In this example, the problem would be purely a selection on observables problem, and there won’t be complications related to unobserved latent confounders -- the goal is to solely demonstrate the step-by-step approach with explicitly stating and showing the causal assumptions being made.
```
Comment (timothyb0912) [inline notebooks/_03-Vision.md:24]: So if I'm understanding correctly, the first part is simply showcasing a causal graph for a realistic simulation example, in the context of selection on observables.

Will we deal with variable selection here or no? We don't need to deal with it, I'm just raising the question.

Variable selection is probably the only thing that computationally will differ in the selection on observables setting.

Comment (timothyb0912) [reply notebooks/_03-Vision.md:24]: For line 26, where it says "(Maybe) apply this approach to a real world dataset as well," why maybe? It seems that this is the most externally valued part of the project.

Comment (timothyb0912) [reply notebooks/_03-Vision.md:24]: Lastly, for part 2, is there also going to be a comparison to a traditional DCM approach for omitted variables? It wasn't specified.

**Review by timothyb0912 — COMMENTED** [review]
Do we know how long the presentation is? That will influence things like expected slide count and how much space we have to work with.

File: notebooks/_05-Requirements.md [hunk 4]
```diff
@@ -42,7 +42,7 @@ jupyter:
     - Output: Simulated variables
```
Comment (timothyb0912) [inline notebooks/_05-Requirements.md:43]: Please include:
- how much time we have for the presentation
- expected slide count range [min, max], based on how much time we have for the presentation.

To come up with this range, think of allocating 1 minute per slide for pure image slides and 3 minutes per slide that requires detailed explanation. Try to give a rough estimate of how much explanation of concepts will need to be done versus visual presentation of results or visual demonstration of concepts.

Comment (timothyb0912) [reply notebooks/_05-Requirements.md:43]: Include tests as as requirement along with inputs and outputs.

Comment (timothyb0912) [reply notebooks/_05-Requirements.md:43]: They require causal graphs as inputs to the process too =]

Comment (timothyb0912) [reply notebooks/_05-Requirements.md:43]: Comparisons across methods (causal inference vs traditional DCM) will be done as part of this project. Comparisons along what metrics?

Comment (timothyb0912) [reply notebooks/_05-Requirements.md:43]: I would include plots with posterior-predictive-p-values

Comment (timothyb0912) [reply notebooks/_05-Requirements.md:43]: Please explicitly note:
- the sort of results we expect to present for each simulation and real data example. Be explicit about the
  - goal of each each simulation result we are showcasing. What do we want the viewer to take away?
  - metrics for each result.
  - presentation of metric (bar cart?, table?, infographic?)
  - appropriateness of the metric + presentation. Does it directly showcase what we want the viewer to take away?

File: notebooks/_01-ProblemStatement.md (hunk 1 above)
Comment (bouzaghrane) [reply notebooks/_01-ProblemStatement.md]: @timothyb0912 @hassanobeid1994  I am thinking something along the lines of: 

The existing disconnect between the fields of travel demand modeling and causal inference make it hard for cross-fertilization between the two fields to take place. This project is motivated by the current lack of use of methods and findings from causal inference in travel demand modeling. 

Thoughts?

File: notebooks/_03-Vision.md (hunk 3 above)
Comment (bouzaghrane) [reply notebooks/_03-Vision.md:24]: > Lastly, for part 2, is there also going to be a comparison to a traditional DCM approach for omitted variables? It wasn't specified.

@timothyb0912 I think it only makes sense to have the comparison. Just a quick question: do you think it isn't necessary? If so, why?

**Comment by bouzaghrane** [comment]
> Do we know how long the presentation is? That will influence things like expected slide count and how much space we have to work with.

@timothyb0912 Personally not sure, but I'm sure Joan knows. If @hassanobeid1994 haven't had the chance to ask Joan, we can ask her tomorrow during our group mtg.

**Comment by bouzaghrane** [comment]
> Do we know how long the presentation is? That will influence things like expected slide count and how much space we have to work with.

From Hassan's email: 
Presentations are 1 hour long, broken as follows:
  - 20 mins are for presenting
  - 15 mins are for discussants to ask questions
  - 25 mins for general questions

File: notebooks/_05-Requirements.md (hunk 4 above)
Comment (bouzaghrane) [reply notebooks/_05-Requirements.md:43]: From Hassan's email:
Presentations are 1 hour long, broken as follows:

20 mins are for presenting
15 mins are for discussants to ask questions
25 mins for general questions

File: notebooks/_03-Vision.md (hunk 3 above)
Comment (timothyb0912) [reply notebooks/_03-Vision.md:24]: Alright, resolving convo based on video call: We'll mention DCM approaches to omitted variables in the talk but not show a comparison in the presentation.

**Pull request merged**